rtic-sync/CHANGELOG.md

```diff
@@ -7,7 +7,10 @@ For each category, _Added_, _Changed_, _Fixed_ add new entries at the top!
 
 ## [Unreleased]
 
+## v1.3.2 - 2025-03-16
+
 ### Fixed
+
 - Improve handling of free slots for `send` by explicitly writing the free slot to the awoken future.
```

Comment:
Forgot to write a better note here: could you add a ref to "fixing all known instances of #780"?

Reply:
Done